### PR TITLE
Add support of Jetty 12.1 ee11

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

#### Changes proposed in this pull request:

-
-
-

**Fixes**: #

### CODE REVIEW

File: jetty-embedded-10/src/main/java/org/jboss/arquillian/container/jetty/embedded_10/ArquillianAppProvider.java [hunk 1]
```diff
@@ -98,6 +98,7 @@ protected App createApp(final Archive<?> archive) {
         try {
             if (this.config.isUseArchiveNameAsContext()) {
                 Path tmpDirectory = Files.createTempDirectory("arquillian-jetty");
+                tmpDirectory.toFile().deleteOnExit();
```
Comment (joakime) [inline jetty-embedded-10/src/main/java/org/jboss/arquillian/container/jetty/embedded_10/ArquillianAppProvider.java:101]: Two problems with this.

A) the deleteOnExit doesn't handle directories really well.
B) the deleteOnExit only works on proper JVM shutdown (a Ctrl+C would bypass this cleanup)

I personally wouldn't use File.deleteOnExit for anything.
If this is a directory passed into Jetty, let Jetty clean it up.

File: jetty-embedded-11/src/main/java/org/jboss/arquillian/container/jetty/embedded_11/ArquillianAppProvider.java [hunk 2]
```diff
@@ -101,6 +101,7 @@ protected App createApp(final Archive<?> archive) {
         try {
             if (this.config.isUseArchiveNameAsContext()) {
                 Path tmpDirectory = Files.createTempDirectory("arquillian-jetty");
+                tmpDirectory.toFile().deleteOnExit();
```
Comment (joakime) [inline jetty-embedded-11/src/main/java/org/jboss/arquillian/container/jetty/embedded_11/ArquillianAppProvider.java:104]: Ditto

File: jetty-embedded-12-ee10/src/main/java/org/jboss/arquillian/container/jetty/embedded_12_ee10/ArquillianAppProvider.java [hunk 3]
```diff
@@ -104,6 +104,7 @@ protected App createApp(final Archive<?> archive) {
         try {
             if (this.config.isUseArchiveNameAsContext()) {
                 Path tmpDirectory = Files.createTempDirectory("arquillian-jetty");
+                tmpDirectory.toFile().deleteOnExit();
```
Comment (joakime) [inline jetty-embedded-12-ee10/src/main/java/org/jboss/arquillian/container/jetty/embedded_12_ee10/ArquillianAppProvider.java:107]: Ditto

File: jetty-embedded-10/src/main/java/org/jboss/arquillian/container/jetty/embedded_10/ArquillianAppProvider.java (hunk 1 above)
Comment (olamy) [reply jetty-embedded-10/src/main/java/org/jboss/arquillian/container/jetty/embedded_10/ArquillianAppProvider.java:101]: works fine for me using Linux (probably osx)

**Review by joakime — APPROVED** [review]
Approved, but be aware of the issues around deleteOnExit, consider using a more robust solution with proper feedback to the user when things go wrong.

**Comment by olamy** [comment]
@joakime I have simplified and so do not need anymore this tmp directory to delete.

**Review by joakime — APPROVED** [review]

**Pull request merged**